src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -148,7 +148,12 @@ export const QueryResultPane = () => {
                 resultPaneParent,
                 ribbonRef.current,
             );
-            setMessageGridHeight(availableHeight);
+            if (
+                metadata.tabStates?.resultPaneTab ===
+                qr.QueryResultPaneTabs.Messages
+            ) {
+                messageGridHeight = availableHeight;
```

Comment:
we still need to have `messageGridHeight` as a state variable, otherwise this line won't trigger a re-rendering.